src/CMakeLists.txt

```diff
@@ -95,7 +95,7 @@ target_link_libraries(CemuBin PRIVATE
 )
 
 target_link_libraries(CemuBin PRIVATE CemuAsm)
-target_link_libraries(CemuBin PRIVATE SDL2::SDL2 SDL2::SDL2main) # is SDL2main needed?
+target_link_libraries(CemuBin PRIVATE SDL2::SDL2) # is SDL2main needed?
```

Comment:
Why was SDL2main removed? If you're getting build failures because CMake can't find the target, it is a known issue that affects the latest SDL2 release, see https://github.com/cemu-project/Cemu/pull/75#issuecomment-1232851866

Reply:
I was getting link errors with lld-link because multiple main has been defined

According to the docs, since `SDL_MAIN_HANDLED` is set in `main.cpp` and `SDL_SetMainReady()` is called in our main function we shouldn't need to link SDL2main

Reply:
Ok, thanks for the clarification! I was also unsure if SDL2main was actually needed, hence the comment in CMakeLists.txt. Since you removed the library, could you also remove the comment? Thanks :)

Reply:
sure